.github/workflows/build-pull-request.yml

```diff
@@ -11,8 +11,9 @@ jobs:
   common:
     uses: ./.github/workflows/build-common.yml
     with:
-      # windows smoke tests are slower, and it's rare for only the windows smoke tests to break
-      skip-windows-smoke-tests: false
+      # it's rare for only the openj9 tests or the windows smoke tests to break
+      skip-openj9-tests: true
```

Comment:
Does it make sense to add a label that triggers these tests?

Reply:
I'll send a follow-up PR 👍